app/components/post_list/post_list.ios.js

```diff
@@ -72,9 +74,26 @@ export default class PostList extends PostListBase {
             ) {
                 this.props.onLoadMoreUp();
             }
+        } else if (pageOffsetY < 0) {
+            if (this.state.postListHeight > contentHeight || this.props.location === 'thread') {
+                // Posting a message like multiline or jumbo emojis causes the FlatList component for iOS
+                // to render RefreshControl component and remain the space as is when it's unmounted,
+                // leaving a whitespace of ~64 units of height between input box and post list.
+                // This condition explicitly pull down the list to recent post when pageOffsetY is less than zero,
+                // and the height of the layout is greater than its content or is on a thread screen.
+                this.handleScrollToRecentPost();
+            }
         }
     };
 
+    handleScrollToRecentPost = debounce(() => {
+        this.refs.list.scrollToIndex({
+            animated: true,
+            index: 0,
+            viewPosition: 0.5,
```

Comment:
this will scroll the index to the center of the screen, I doubt this is what you wanted here

Reply:
Looks working fine while testing, maybe because of 0 index :).  Corrected now to 1.

Reply:
I think you made the change in the wrong place

Reply:
Sorry for that, my bad.
Now updated, I've tested/verified the changes and it works fine on me.